chainer/functions/pooling/roi_align_2d.py

```diff
@@ -540,6 +568,8 @@ def roi_align_2d(x, rois, outh, outw, spatial_scale, sampling_ratio=0):
             It must meet >=0 and is automatically decided when 0 is passed.
             Use of different ratio in height and width axis is also supported
             by passing tuple of int as (sampling_ratio_h, sampling_ratio_w).
+        inner_bounds (bool): Bounding box represents inner bounds or not.
+            Default is `False`.
```

Comment:
Can we use str instead of bool to represent the bounds mode, like `'inner'` and `'outer'`? (I thought a bool flag should be used when the argument represents on/off of a feature, but the case here is not like that)

Reply:
Updated. I modified `bounds` and check the input.